build-hazelcast-homebrew-package.sh

```diff
@@ -55,7 +55,7 @@ function generateFormula {
   updateClassName "$class" "$file"
   sed -i "s+url.*$+url \"${HZ_PACKAGE_URL}\"+g" "$file"
   sed -i "s+sha256.*$+sha256 \"${ASSET_SHASUM}\"+g" "$file"
-  all_hz_versions=({hazelcast.rb,hazelcast-devel.rb,hazelcast-snapshot.rb,hazelcast?[0-9]*\.rb,hazelcast-enterprise*\.rb})
+  all_hz_versions=({hazelcast.rb,hazelcast?[0-9]*\.rb,hazelcast-enterprise*\.rb})
   for version in "${all_hz_versions[@]}"
   do
     if [[ "$version" != "$file" ]] && [[ ! "$version" =~ .*(beta|devel).* ]] ; then
```

Comment:
Do we need `devel` as removed above?
Mind you don't have `beta` in `all_hz_versions`
tbh not sure I fully understand this hence the question

Reply:
> Do we need `devel` as removed above?


I already deleted them from the repo so I hope not

 https://github.com/hazelcast/homebrew-hz/commit/1e29665d95ec8e80cfffe5c7b0445e78744d3efd

Reply:
Sure.
we still have this https://github.com/hazelcast/homebrew-hz/blob/master/hazelcast-enterprise-devel.rb (2 years ago)
We don't build Packaging for devel. Just mentioning and not saying you do anything

My question is related to `if [[ "$version" != "$file" ]] && [[ ! "$version" =~ .*(beta|devel).* ]] ; then`

We don't Package DEVEL and as its now been removed from `all_hz_versions`, is there any need to check for `devel` in the regex?

Reply:
@ldziedziul

Do we need to check for `devel` anymore in `[[ ! "$version" =~ .*(beta|devel).* ]]?:`

- `devel` is being removed from `all_hz_versions` (and the for loop is traversing over the array)
- we don't do Packaging for `devel`

Reply:
@nishaatr if we don't and won't have devel packages `devel` check is not needed

Reply:
I can remove `DEVEL`, but it's in other places, too:
https://github.com/hazelcast/hazelcast-packaging/blob/802de781c5c1df8091c3d7d24d24c7c2023c71a4/common.sh#L12

Reply:
@nishaatr is skipping packaging `devel` is permanent?

Reply:
I think its always been like this since I started working. We don't use or tag packaging https://github.com/hazelcast/hazelcast-mono/blob/master/.github/workflows/development-release.yml
And there are no DEVEL tags in the repo. May be we used to package a while back and at some point it was removed 
As DEVEL is internal Docker is sufficient I believe 

So I think we can cleanup DEVEL completely. I will raise PR to delete 

@JackPGreen i think best we do this in separate PR as need to cleanup Jfrog and GH as well.
https://hazelcast.jfrog.io/ui/repos/tree/General/rpm-local/devel
https://hazelcast.jfrog.io/ui/repos/tree/General/rpm-test-local/devel
https://hazelcast.jfrog.io/ui/repos/tree/General/rpm/devel (might be a pointer to rpm-local)
https://github.com/hazelcast/homebrew-hz/blob/master/hazelcast-enterprise-devel.rb

I don't mind doing it so let me know!

Reply:
> So I think we can cleanup DEVEL completely. I will raise PR to delete
> 
> I don't mind doing it so let me know!

Thanks, separate is best.

Reply:
FYI https://hazelcast.atlassian.net/browse/DI-241